web/www/horas/Latin/Sancti/12-27.txt

```diff
@@ -225,23 +233,23 @@ $Deo gratias
 Exiit * sermo inter fratres, quod discipulus ille non moritur; et non dixit Jesus, Non moritur: sed, Sic eum volo manere, donec veniam.
 
 [Octava 2]
-!Commemoratio octavae S. Joannis.
-Ant. Iste est Joannes, qui supra pectus Domini in cena recubuit: beatus Apostolus, cui revelata sunt secreta caelestia.
+!Commemoratio Octavae S. Joannis
+Ant. Iste est Joannes, qui supra pectus Domini in caena recubuit: beatus Apostolus, cui revelata sunt secreta coelestia.
```

Comment:
At least as far as the 1960 rubrics are concerned, `Ant 2` should to be corrected to _cena_ and _cælestia,_ and also in `Responsory8` this spelling should be followed. The 1961 _editio typica_ is the official model in these cases. If `Ant 2` is going to continue to be `@:Ant 1`, then the change has to be made in `Ant 1`.

As for the Commemoratio, there isn't an official model I'm aware of, and I don't personally have any authority to make decisions for the Divinum Officium Project, but I think following the same style throughout the database of texts is the simplest practice.